### DISCUSSION TITLE
Technics 1210 -> Audio4DJ -> OBS

### DISCUSSION THREAD

**Discussion by turntablestreamer** [ExistentialAudio/BlackHole#83] [General]
Hi,

My problem is following - I’d like to stream music from Technics 1210 turntables via OBS using mixer plugged into external sound card Audio4DJ with USB connection (as my Macbook doesn’t allow conversion of the "headphones" jack).
I installed the latest BlackHole for Catalina according to the manual and whichever combination I did (in my Audio MIDI settings, with Multi-Output Device or in OBS) the OBS doesn’t receive and send any signal.

I also tried to follow this video, but without any success at the end:
https://www.youtube.com/watch?v=miexiUIusts&feature=youtu.be

The only thing that worked so far was the competitor’s product - Loopback.

Hope the solution for this case exists and thank you for any hints!

EDIT1: When I’m recording music to GarageBand, there is no problem at all with the sound signal.

**Comment by devinroth** [comment]
BlackHole passes the output to the input. In order to do what you want you need something to pass the input to the output. Just use garage band to help you out. 

Audio in to Garage Band. Garage Band output to BlackHole. BlackHole input to OBS.

**Comment by turntablestreamer** [comment]
devinroth: Well... and is this possible? To send signal from GrageBand, which is recording app, to BlackHole? If yes, how?

**Comment by devinroth** [comment]
I know in Logic Pro I can set a different input and output device. GarageBand should be similar. But if not try using an aggregate device. Route your input to the BlackHole output.

**Comment by devinroth** [comment]
Are you able to just go directly to OBS?
